chat.js

```diff
@@ -1186,7 +1186,9 @@ Chat.getDataMoveHTML = function (move) {
 	buf += `<img src="//play.pokemonshowdown.com/sprites/categories/${move.category}.png" alt="${move.category}" width="32" height="14"></span> `;
 	if (move.basePower) buf += `<span class="col labelcol"><em>Power</em><br>${typeof move.basePower === 'number' ? move.basePower : '—'}</span> `;
 	buf += `<span class="col widelabelcol"><em>Accuracy</em><br>${typeof move.accuracy === 'number' ? (move.accuracy + '%') : '—'}</span> `;
-	buf += `<span class="col pplabelcol"><em>PP</em><br>${move.pp ? move.pp : 1}</span> `;
+	const basePP = move.pp || 1;
+	const pp = basePP === 1 || move.noPPBoosts ? basePP : basePP * 8 / 5;
```

Comment:
Maybe it should be floored instead of `basePP === 1`. That seems to more closely match how PP Ups work.